js/windowEventRecorder.js

```diff
@@ -28,9 +28,10 @@ const recordEvent = (e) => {
 
   const attrFilter = [
     'class',
-    'id'
+    'id',
+    'style'
   ];
```

Comment:
Found a little bug here. I saw a selector such as `[style="background: url(...some-image-path.png)"]`, which is not valid. Let's ignore the `style` attribute as well

Reply:
Which is also often dynamically generated..

Reply:
Yeah, good call. I have seen recordings show up using a style tag and it always looked messy and brittle.